src/components/publiq-ui/Spinner.js

```diff
@@ -0,0 +1,39 @@
+import PropTypes from 'prop-types';
+import { Spinner as BootstrapSpinner } from 'react-bootstrap';
+import styled from 'styled-components';
+
+const SpinnerVariants = {
+  PRIMARY: 'primary',
+  LIGHT: 'light',
+};
+
+const SpinnerSizes = {
+  NORMAL: undefined,
```

Comment:
undefined will result in the default spinner size?

Reply:
jep

Reply:
not completely clear why we're doing this? we have a defaultProp right?

Reply:
I've fixed it, better?